docs/sql/functions/aggregates.md

```diff
@@ -107,10 +107,12 @@ The table below shows the available general aggregate functions.
 |:--|:--------|
 | [`any_value(arg)`](#any_valuearg) | Returns the first non-null value from `arg`. This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
 | [`arbitrary(arg)`](#arbitraryarg) | Returns the first value (null or non-null) from `arg`. This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
-| [`arg_max(arg, val)`](#arg_maxarg-val) | Finds the row with the maximum `val`. Calculates the `arg` expression at that row. This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
-| [`arg_max(arg, val, n)`](#arg_maxarg-val-n) | Returns a `LIST` containing the `arg` expressions for the "top" `n` rows ordered by `val` descending. |
-| [`arg_min(arg, val)`](#arg_minarg-val) | Finds the row with the minimum `val`. Calculates the `arg` expression at that row. This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
-| [`arg_min(arg, val, n)`](#arg_minarg-val-n) | Returns a `LIST` containing the `arg` expressions for the "bottom" `n` rows ordered by `val` ascending.  |
+| [`arg_max(arg, val)`](#arg_maxarg-val) | Finds the row with the maximum `val` and calculates the `arg` expression at that row. Rows where the value of the `arg` or `val` expression is `NULL` are ignored. This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
+| [`arg_max(arg, val, n)`](#arg_maxarg-val-n) | The generalized case of [`arg_max`](#arg_maxarg-val) for `n` values: returns a `LIST` containing the `arg` expressions for the top `n` rows ordered by `val` descending. This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
+| [`arg_max_null(arg, val)`](#arg_max_nullarg-val) | Finds the row with the maximum `val` and calculates the `arg` expression at that row. Rows where the `arg` expression evaluates to `NULL` are ignored.  This function is [affected by ordering](#order-by-clause-in-aggregate-functions). |
```

Comment:
I think the `NULL` behavior is actually reversed. Rows where the `arg` expression evaluates to `NULL` are _not_ ignored (rows where the `val` expression evaluates to `NULL` are ignored). e.g.
```sql
D create or replace table foo (arg text, val integer);
D insert into foo values ('a', 4), ('e', null), (null, 9), ('b', 5);
D select * from foo;
┌─────────┬───────┐
│   arg   │  val  │
│ varchar │ int32 │
├─────────┼───────┤
│ a       │     4 │
│ e       │       │
│         │     9 │
│ b       │     5 │
└─────────┴───────┘
D select arg_max(arg, val) from foo;
┌───────────────────┐
│ arg_max(arg, val) │
│      varchar      │
├───────────────────┤
│ b                 │
└───────────────────┘
D select arg_max_null(arg, val) from foo;
┌────────────────────────┐
│ arg_max_null(arg, val) │
│        varchar         │
├────────────────────────┤
│                        │
└────────────────────────┘
```